README.md

```diff
@@ -1 +1,27 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+## Getting Started
+
+### Setup
+
+### Usage
```

Comment:
- [ ] Hi, great work following suggestions from the previous reviewer. You have a nice README for the most part 👍. I would like you to add details about how to setup up the project and a small description of the project. Or remove headings that do not have anything under them 
Something as simple as this would make a much better README
```suggestion
## Getting Started
to set this project up please run these commands:
`git clone git@github.com:OybekKayumov/Hello-Microverse.git`
`cd Hello-Microverse`
```

Reply:
Hello, Alonso!
I think I have done all the changes.
Please, check my project.
best regards,
Oybek Kayumov